pkg/kube/controllers/extendedjob/errand_reconciler.go

```diff
@@ -34,6 +34,8 @@ const (
 	EnvCfOperatorAzIndex = "AZ_INDEX"
```

Comment:
Can you please rename this to `EnvCFOperatorAZIndex`?

Reply:
Done.